03_qgep_db_dss.sql

```diff
@@ -3923,7 +3923,7 @@ ALTER TABLE qgep.vl_structure_part_renovation_demand ADD CONSTRAINT pkey_qgep_vl
  REFERENCES qgep.vl_structure_part_renovation_demand (code) MATCH SIMPLE 
  ON UPDATE RESTRICT ON DELETE RESTRICT;
 ALTER TABLE qgep.od_structure_part ADD COLUMN fk_wastewater_structure varchar (16);
-ALTER TABLE qgep.od_structure_part ADD CONSTRAINT rel_structure_part_wastewater_structure FOREIGN KEY (fk_wastewater_structure) REFERENCES qgep.od_wastewater_structure(obj_id);
+ALTER TABLE qgep.od_structure_part ADD CONSTRAINT rel_structure_part_wastewater_structure FOREIGN KEY (fk_wastewater_structure) REFERENCES qgep.od_wastewater_structure(obj_id) ON DELETE set null;
```

Comment:
Hi @m-kuhn 
It is the other way round. When I delete a cover (which is a structure part) I do not want the wastewater structure to be deleted also. E.g. I delete a second cover or just a benching or ladder. So I think it is correct this way. This points from structure_part -> wastewater structure, not the other way round, then it would be wrong